packages/client/hmi-client/src/components/widgets/tera-import-github-file.vue

```diff
@@ -334,8 +335,9 @@ async function importDataFiles(githubFiles: GithubFile[]) {
 		await createNewDatasetFromGithubFile(
 			repoOwnerAndName.value,
 			githubFile.path,
-			props.project?.username ?? '',
-			props.project?.id ?? ''
+			auth.name ?? '',
```

Comment:
I think those should be done on the HMI-server, I don't see the need to send that information from the front-end.
What do you think?